packages/storefront/src/pages/developing/angular/advanced.md

```diff
@@ -2,6 +2,61 @@
 
 <TableOfContents></TableOfContents>
 
+## Global Theme
+
+**Since v3.8.0** you can set the `theme` property for all child components by passing an option to
```

Comment:
Shouldn't this be the next version released here? Since we are currently on 3.8.0